spec/site_prism/page_objects/page/organisations/organisation_page.rb

```diff
@@ -6,8 +6,8 @@ class OrganisationPage < PageObjects::Base
 
         element :locations, "[data-qa=provider__locations]", text: "Locations"
         element :courses, "[data-qa=provider__courses]", text: "Courses"
-        element :current_cycle, "[data-qa=provider__courses__current_cycle]", text: "Current cycle (2019 - 2020)"
-        element :next_cycle, "[data-qa=provider__courses__next_cycle]", text: "Next cycle (2020 - 2021)"
+        element :current_cycle, "[data-qa=provider__courses__current_cycle]", text: "New cycle (2020 – 2021)"
```

Comment:
will `New` be filpped back to `Current`?

Reply:
Probably from Oct 1st. This design needs more work